raiden/ui/cli.py

```diff
@@ -632,6 +633,7 @@ def print_step(description: str, error: bool = False) -> None:
         port = next(free_port_generator)
 
         args["api_address"] = "localhost:" + str(port)
+        args["routing_mode"] = RoutingMode.PRIVATE
```

Comment:
This is for testing purposes only? Or we're setting this value always in Private for some reason?